{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/pulsecore\/loggerstore\/error\/errordescription-7jg1j"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/PulseCore\/documentation\/PulseCore\/LoggerStore\/Error\/errorDescription-7jg1j","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"LocalizedError.errorDescription"},{"type":"text","text":"."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"title":"errorDescription","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"s:9PulseCore11LoggerStoreC5ErrorO16errorDescriptionSSSgvp","modules":[{"name":"PulseCore"}]},"hierarchy":{"paths":[["doc:\/\/PulseCore\/documentation\/PulseCore","doc:\/\/PulseCore\/documentation\/PulseCore\/LoggerStore","doc:\/\/PulseCore\/documentation\/PulseCore\/LoggerStore\/Error"]]},"references":{"doc://PulseCore/documentation/PulseCore/LoggerStore":{"role":"symbol","title":"LoggerStore","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"LoggerStore"}],"abstract":[{"type":"text","text":"Persistently stores logs, network requests, and response blobs."}],"identifier":"doc:\/\/PulseCore\/documentation\/PulseCore\/LoggerStore","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"LoggerStore"}],"url":"\/documentation\/pulsecore\/loggerstore"},"doc://PulseCore/documentation/PulseCore":{"role":"collection","title":"PulseCore","abstract":[{"type":"text","text":"Logger and network inspector for Apple platforms."}],"identifier":"doc:\/\/PulseCore\/documentation\/PulseCore","kind":"symbol","type":"topic","url":"\/documentation\/pulsecore"},"doc://PulseCore/documentation/PulseCore/LoggerStore/Error/errorDescription-7jg1j":{"role":"symbol","title":"errorDescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/PulseCore\/documentation\/PulseCore\/LoggerStore\/Error\/errorDescription-7jg1j","kind":"symbol","type":"topic","url":"\/documentation\/pulsecore\/loggerstore\/error\/errordescription-7jg1j"},"doc://PulseCore/documentation/PulseCore/LoggerStore/Error":{"role":"symbol","title":"LoggerStore.Error","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"Error"}],"abstract":[],"identifier":"doc:\/\/PulseCore\/documentation\/PulseCore\/LoggerStore\/Error","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Error"}],"url":"\/documentation\/pulsecore\/loggerstore\/error"}}}